.ci/watchdog/Watchdog.py

```diff
@@ -424,7 +439,10 @@ def _queue_message(self, message, message_severity='info', pr=None):
             message = message + '\n' + pr.html_url
 
         send = message_header + '\n' + message
-        self._slack_app.queue_message(send, internal_error=internal)
+        if self._slack_enabled:
+            self._slack_app.queue_message(send, internal_error=internal)
+        if self._ms_teams_enabled:
```

Comment:
It would be less error-prone to add a general communicator class/object that would abstract sending messages thtough chosen applications. Maybe as future TODO, if You want to enable this fast?